Dockerfile

```diff
@@ -25,4 +46,4 @@ LABEL org.opencontainers.image.licenses="Apache-2.0+GPL-2.0-only"
 LABEL org.opencontainers.image.vendor="Arcalot project"
 LABEL org.opencontainers.image.authors="Arcalot contributors"
 LABEL org.opencontainers.image.title="Python Plugin Template"
-LABEL io.github.arcalot.arcaflow.plugin.version="1"
+LABEL io.github.arcalot.arcaflow.plugin.version="1"
```

Comment:
Now we are missing a newline.

Reply:
accidental delete of both. I have added one back